content/en/docs/concepts/storage/persistent-volumes.md

```diff
@@ -71,9 +71,9 @@ A user creates, or in the case of dynamic provisioning, has already created, a P
 
 Claims will remain unbound indefinitely if a matching volume does not exist. Claims will be bound as matching volumes become available. For example, a cluster provisioned with many 50Gi PVs would not match a PVC requesting 100Gi. The PVC can be bound when a 100Gi PV is added to the cluster.
 
-### Using
+### Mounting and Using {#using}
 
-Pods use claims as volumes. The cluster inspects the claim to find the bound volume and mounts that volume for a Pod. For volumes that support multiple access modes, the user specifies which mode is desired when using their claim as a volume in a Pod.
+Pods use claims as volumes. The cluster inspects the claim to find a matching PersistentVolume for the Pod, and to check that the Pod is entitled to use it. If the claim is valid and the storage is available, the kubelet sets up storage access so that the PersistentVolume is available to containers in the Pod (either as mounted storage or as a block device). The PersistentVolumeClaim also defines whether the kubelet should mount the volume read-only or read-write.
```

Comment:
I think this part "find the bound volume and mounts that volume for a Pod" should still be mentioned.

Reply:
CC @msau42

Reply:
Hello @Colstuwjx .
Are you able to add back the original sentence or is there a question about `the Pod is entitled to use it`?
Ping @msau42 , Would you have time to review the changes? Thank you.